services/src/main/java/org/keycloak/services/resources/admin/AdminRoot.java

```diff
@@ -223,6 +224,12 @@ public Object getRealmsAdmin() {
         return new RealmsAdminResource(session, auth, tokenManager);
     }
 
+    @Path("{any:.*}")
+    @OPTIONS
+    public Object preFlight() {
+        return getRealmsAdmin();
+    }
```

Comment:
Maybe to make it more readable and straightforward? Even though the result is the same, I think we should not be returning `/realms` for `{any:.*}`.
```suggestion
    @Path("{any:.*}")
    @OPTIONS
    public Object preFlight() {
        return new AdminCorsPreflightService(request);
    }
```

Also, I think we can remove these two:
https://github.com/keycloak/keycloak/blob/b65b1fddf3ea8f09251161b0573519cd9205a494/services/src/main/java/org/keycloak/services/resources/admin/AdminRoot.java#L210-L212
https://github.com/keycloak/keycloak/blob/b65b1fddf3ea8f09251161b0573519cd9205a494/services/src/main/java/org/keycloak/services/resources/admin/AdminRoot.java#L248-L250

Reply:
This is also a limitation related to Resteasy Reactive. I can send in another PR.